backend/src/test/java/com/woowacourse/f12/documentation/MemberDocumentation.java

```diff
@@ -0,0 +1,108 @@
+import org.springframework.boot.test.mock.mockito.MockBean;
+import org.springframework.http.HttpHeaders;
+import org.springframework.http.MediaType;
+import org.springframework.test.web.servlet.MockMvc;
+import org.springframework.test.web.servlet.ResultActions;
+
+@WebMvcTest(MemberController.class)
+public class MemberDocumentation extends Documentation {
+
+    @Autowired
+    private MockMvc mockMvc;
+
+    @MockBean
+    private MemberService memberService;
+
+    @MockBean
+    private JwtProvider jwtProvider;
+
+    private final ObjectMapper objectMapper = new ObjectMapper();
+
+    @Test
+    void 로그인된_상태에서_나의_회원정보를_조회_API_문서화() throws Exception {
+        // given
+        String authorizationHeader = "Bearer Token";
+        given(jwtProvider.validateToken(authorizationHeader))
+                .willReturn(true);
+        given(jwtProvider.getPayload(authorizationHeader))
+                .willReturn("1");
+        given(memberService.findById(1L))
+                .willReturn(MemberResponse.from(CORINNE.생성(1L)));
+
+        // when
+        ResultActions resultActions = mockMvc.perform(
+                get("/api/v1/members/me")
+                        .header(HttpHeaders.AUTHORIZATION, authorizationHeader)
+        );
+
+        // then
+        resultActions.andExpect(status().isOk())
+                .andDo(document("members-get-mine"))
+                .andDo(print());
+    }
+
+    @Test
+    void 비로그인_상태에서_회원정보를_조회_API_문서화() throws Exception {
+        // given
+        Long memberId = 1L;
+        given(memberService.findById(memberId))
+                .willReturn(MemberResponse.from(CORINNE.생성(memberId)));
+
+        // when
+        ResultActions resultActions = mockMvc.perform(
+                get("/api/v1/members/" + memberId)
+        );
+
+        // then
+        resultActions.andExpect(status().isOk())
+                .andDo(document("members-get-by-memberId"))
+                .andDo(print());
+    }
+
+    @Test
+    void 로그인된_상태에서_나의_회원정보를_수정_API_문서화() throws Exception {
+        // given
+        MemberRequest memberRequest = new MemberRequest(JUNIOR, BACK_END);
+        String authorizationHeader = "Bearer Token";
+        given(jwtProvider.validateToken(authorizationHeader))
+                .willReturn(true);
+        given(jwtProvider.getPayload(authorizationHeader))
+                .willReturn("1");
+        willDoNothing().given(memberService).updateMember(1L, memberRequest);
+
+        // when
+        ResultActions resultActions = mockMvc.perform(
+                patch("/api/v1/members/me")
+                        .header(HttpHeaders.AUTHORIZATION, authorizationHeader)
+                        .contentType(MediaType.APPLICATION_JSON_VALUE)
+                        .content(objectMapper.writeValueAsString(memberRequest))
+        );
+
+        // then
+        resultActions.andExpect(status().isOk())
+                .andDo(document("members-update-me"))
+                .andDo(print());
```

Comment:
```suggestion
                .andDo(document("members-update-mine"))
                .andDo(print());
```

근데 사실 `mine`들어간 곳 전부 `mine`보다는 `own`이 낫지 않나요?

Reply:
그리고 어차피 update는 남의거를 못할테니까 `me`가 필요 없을 것 같아요